include/walker-dropdown.php

```diff
@@ -19,19 +19,20 @@ class PLL_Walker_Dropdown extends Walker {
 	public $db_fields = array( 'parent' => 'parent', 'id' => 'id' );
 
 	/**
-	 * Outputs one element
+	 * Outputs one element.
 	 *
 	 * @since 1.2
 	 *
-	 * @param string   $output            Passed by reference. Used to append additional content.
-	 * @param stdClass $element           The data object.
-	 * @param int      $depth             Depth of the item.
-	 * @param array    $args              An array of additional arguments.
-	 * @param int      $current_object_id ID of the current item.
+	 * @param string $output            Passed by reference. Used to append additional content.
+	 * @param object $element           The data object.
+	 * @param int    $depth             Depth of the item.
+	 * @param array  $args              An array of additional arguments.
+	 * @param int    $current_object_id ID of the current item.
 	 * @return void
 	 */
 	public function start_el( &$output, $element, $depth = 0, $args = array(), $current_object_id = 0 ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 		$value_type = $args['value'];
+		$element = $this->convert_to_standard_class( $element );
```

Comment:
Could this work?
```php
		$element = (object) (array) $element;
```
Alos could it be sufficient to make teh conversion in `walk()`?

Reply:
Not according to this discussion: https://github.com/polylang/polylang/pull/1195#pullrequestreview-1245609057
If we want to use `PLL_Language::get_object_vars()` we can't do that IMHO.